ui/components/app/nft-details/nft-detail-description.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```

Comment:
Emm I think when i started created nft-details component in js, i finished writing the rest of the components in Js too,
Does this require to convert the rest of the components to tsx too?

Reply:
I updated the new created components to `.tsx`; i kept the `nft-details.js` because when i wanted to update it to tsx and use `Nft` type from core, realized it would require also changing the send.js component.

Reply:
Updated the files to .tsx 🙏